megamek/src/megamek/client/ui/swing/MovementDisplay.java

```diff
@@ -5670,7 +5670,8 @@ private void setSelfDestructEnabled(boolean enabled) {
     }
 
     private void setTraitorEnabled(boolean enabled) {
-        clientgui.getMenuBar().setEnabled(MoveCommand.MOVE_TRAITOR.getCmd(), enabled);
+     // Menu entry does not exist
+//        clientgui.getMenuBar().setEnabled(MoveCommand.MOVE_TRAITOR.getCmd(), enabled);
```

Comment:
Why keep these if they don't exist?

Reply:
Someone thought at some point they should exist. I personally think all the Move/Fire in-game-action menu commands should be removed. That's what the buttons are for. The menu is for loading games, saving maps or other stuff. In this PR I didn't want to adress any of this, just remove the log errors.